crawl-ref/source/describe.cc

```diff
@@ -569,6 +570,8 @@ static string _randart_descrip(const item_def &item)
         { ARTP_FRAGILE, "It will be destroyed if unequipped.", false },
         { ARTP_SHIELDING, "It affects your SH (%d).", false},
         { ARTP_HARM, "It increases damage dealt and taken.", false},
+        { ARTP_RAMPAGING, "Its wearer goes twice as far when moving towards enemies.",
```

Comment:
```suggestion
        { ARTP_RAMPAGING, "It makes you to go goes twice as far when moving towards enemies.",
```

These are all second person or passive voice, no third-person wearers.

Reply:
Yeah, that wording was a copy of the rampage description added in 50c865f. If we're updating it to match the phrasing of the rest of the artprops, I'd prefer to rephrase both rampage descriptions slightly to remove some potential spoilers. Something like:

SPARM: "Its wearer takes one free step when moving towards enemies."
ARTP: "It bestows one free step when moving towards enemies."

Reply:
I've tweaked both of these descriptions, squashed to reword the commit message, and force pushed with the updated descriptions.